spec/support/draft_helpers.rb

```diff
@@ -185,7 +185,7 @@ def add_related_urls(type: nil, button_type: nil)
               fill_in 'URL', with: 'https://example.com/'
 
               # Get Data fields
-              select 'ascii', from: 'Format'
```

Comment:
explanation: `'ascii'` is not contained in the 'test' KMS keywords response (which is necessary to select it from the picklist as is done in the spec)